config/localization/en/app.json

```diff
@@ -1556,6 +1558,7 @@
       "MSG_CREATE_PERPETUAL": "Msg create perpetual",
       "MSG_DELAY_MESSAGE": "Msg delay message",
       "MSG_SUBMIT_PROPOSAL": "Msg submit proposal",
+      "NEW_MARKET_PROPOSAL_AGREEMENT": "By checking the below box, you acknowledge and understand that (a) the pre-populated list of markets does not reflect an endorsement of any particular asset or market, and should not be relied on as investment, legal or any other form of professional advice; (b) this list is not a recommendations of any specific market, and markets are included based on compatibility and functionality assessments from a technical standpoint using public data; and (c) you are encouraged to conduct your own research and consult qualified legal counsel to ensure compliance with the laws of any and all applicable jurisdictions. Additionally, you acknowledge and understand that by clicking “Propose new market”, you will create an on-chain governance proposal — more information about on-chain governance proposals can be found on dYdX Foundation’s documentation {DOCUMENTATION_LINK}, which is independent from and unaffiliated with the host and operator of this website. Use of this widget is prohibited in the U.S., Canada and sanctioned jurisdictions as described in the {TERMS_OF_USE}",
```

Comment:
maybe typos?

by checking "the box below"
this list is not a "recommendation" 
period at the end?

Reply:
Fixed all except for period at the end.